part2/countries/src/App.js

```diff
@@ -0,0 +1,122 @@
+
+import React, { useState, useEffect } from 'react'
+import axios from 'axios'
+
+const Filter = ({ handleFilterCountry }) => (
+  <>
+    <p> Find countries </p>
+    <input onChange={handleFilterCountry} />
+  </>
+)
+
+const Language = ({ language }) => (
+  <li>{language}</li>
+)
+
+
+const Country = ({ country }) => {
+  const [weather, setWeather] = useState([])
+  // const api_key = process.env.REACT_APP_API_KEY
+
+
+  useEffect(() => {
+    axios
+      .get(`http://api.weatherapi.com/v1/current.json?key=${process.env.REACT_APP_API_KEY}&q=${country.capital}`)
+      .then(response => {
+        setWeather(response.data.current.condition)
+      })
+  }, [country])
+  console.log(weather.current)
+  return (
+    <>
+      <h3>{country.name}</h3>
+      <p>Capital: {country.capital}</p>
+      <p>Population: {country.population}</p>
+      <h3>Language(s)</h3>
+
+      <ul>
+        {country.languages.map((language) =>
+          <Language key={language.name} language={language.name} />
+        )}
+      </ul>
+
+      <img alt="flag" src={country.flag} />
+
+      <h3>Weather in {country.capital}</h3>
+      <div>{weather.text}</div>
+      <img alt="flag" src={weather.icon} />
+    </>
+  )
+}
+
+const List = ({ countriesList, setToCountry }) => {
+  // joprojām šis if neiet cauri
+  if (countriesList.length > 10 && !countriesList.length === 1) {
```

Comment:
Nepareizs lietojums otrajā expressionā. Tādēļ feilo.

Reply:
```
else if (countriesList.length === 1) {
    return (
      <ul>
        {countriesList.map((country) =>
          <Country key={country.name} country={country} />
        )}
      </ul>
```
mapo un prasa sarakstu, lai gan vajag uzrādīt tikai vienu valsti? Ja noņemu šo un ielieku vienu Country kompenenti, man nobrūk Country komponentes Languages - nebūs te vairs nekādas mapošanas, man saka.

Reply:
Ā, vispār tas arī. 👍  Es no telefona reviewoju, neizgāju cauri algoritmam.

Es vairāk par to, ka tas expressions nedara to ko tu gribi, jo izsaukumzīme ir ne tur kur vajag.

Reply:
Jā, es procesā pārvietoju to izsaukuma zīmi : **!==** , bet beigās, izskatās, ka nemaz nevajag